swagger/pelican-swagger.yaml

```diff
@@ -476,7 +476,19 @@ definitions:
       type:
         type: string
         example: "posix"
-        description: "The storage type of the origin server. Is either \"posix\" or \"s3\""
+        description: "The storage type of the origin server. Values can be posix, s3, and https"
+      s3Region:
+        type: string
+        description: "The server region for AWS S3 storage. Only available when type == s3"
+      s3ServiceUrl:
+        type: string
+        description: "The URL that provides API access the the S3 objects. Only available when type == s3"
+      s3UrlStyle:
+        type: string
+        description: "The style of S3 urls used by the service URL host. Can be either \"path\" or \"virtual\". Only available when type == s3"
```

Comment:
Is it worth providing an example of the difference between the two here, or are you happy with what's in our docs elsewhere?

Reply:
I think this is mostly a rough intro for people looking up the API for what they'll get out of that endpoint. If they are interested in learning what exactly the value is for, they can refer to the docs website. I'll link to it